earth_enterprise/src/installer/install_fusion.sh

```diff
@@ -677,6 +679,9 @@ setup_fusion_daemon()
 	# setup fusion daemon
 	printf "Setting up the Fusion daemon...\n"
 
+    # Create a new file ‘/etc/init.d/gevars.sh’ and add the below lines.
```

Comment:
Indentation mismatch here due to spaces vs hard tabs.
Here string might look prettier, but this will work.

Reply:
So I'd say fix the indentation, but it's up to you on keeping this perfectly good command that's already been tested or switching it out and waiting for re-testing.